backend/storage/schema.sql

```diff
@@ -229,3 +229,18 @@ CREATE TABLE wallets (
 );
 
 CREATE INDEX wallets_by_account ON wallets (account);
+
+-- Stores text content to to a full text search
+-- https://sqlite.org/fts5.html.
+
+CREATE VIRTUAL TABLE fts USING fts5(
+    -- The text content to be indexed.
+    raw_content, 
+    -- The type of the content being indexed.
+    type, 
+    -- The multihash of the blob containting the content.
+    multihash,
```

Comment:
@juligasa few questions:

1. Is there a way to have indexes on columns of FTS5 tables?
2. Any particular reason we use multihash here instead of the internal integer ID, or full CID?

One of the concerns I have is that we might need to clean up the FTS5 table after we've deleted a blob (which we don't currently do, but I bet implementing GC is not too far away).

Reply:
Actually, I just remembered that it was one of the reasons I wanted to use FTS5 with external content table: a document ID in FTS5 can only be a rowid (IIRC), and our blobs table have rowids already. So some of the cleanup bookkeeping could be easier to implement this way.

Reply:
I changed it so that we store the blobID instead. The idea is doing an incremental inplementation, First a quick full end to end full text search and then optimizations.